python/packages/autogen-core/docs/src/reference/index.md

```diff
@@ -46,7 +46,7 @@ python/autogen_ext.agents.file_surfer
 python/autogen_ext.agents.video_surfer
 python/autogen_ext.agents.video_surfer.tools
 python/autogen_ext.models
```

Comment:
let's expand this module into submodules like other ones.

```
python/autogen_ext.models.openai
```

Reply:
Updated. I was unsure if we should keep it at the models level because the `ReplayChatCompletionClient`. If we expand it to submodules wont it be left out of the docs?

Reply:
Right I forgot about it. It should be in its own module as well.

Reply:
To close this one. 

1. Do I keep `ReplayChatCompletionClient`  at  `autogen_ext.models`  or move it? If I move it, any idea of what should be namespace? `base`? I can't think of anything. I wouldnt have an issue if it stays at `autogen_ext.models`  as there is no clear `extra`  need for it. It should be fine as long as we don't add the other packages to the `__init__.py`
2. If we keep it at `autogen_ext.models`, do we need both `autogen_ext.models`  and `autogen_ext.models.openai`  in the docs? I thought the former with `:members:` would automatically include the submodules but I'm unsure if that is true if they are not present in the init.

Reply:
You got me thinking... My initial thought was it should just go into `autogen_ext.models.mock` or `testing` or something like that. @jackgerrits what's your thought?

Reply:
`mock` or `replay` makes sense to me

Reply:
Done